packages/gatsby-image/src/index.js

```diff
@@ -464,6 +487,8 @@ Image.propTypes = {
   critical: PropTypes.bool,
   style: PropTypes.object,
   imgStyle: PropTypes.object,
+  objectFit: PropTypes.string, // needs to be separate for object-fit-image polyfill's font-family declaration
+  objectPosition: PropTypes.string, // needs to be separate for object-fit-image polyfill's font-family declaration
```

Comment:
I don't think that comment is needed here as well - this is not only for polyfills

Reply:
I’m wondering if devs might think they can just pass these properties to imgStyle (which they can, but then the polyfill will silently fail)...

Any tips for making it clear that the obj declarations need to be passed outside of imgStyle to enable polyfilling?

Reply:
I reworded the comment to make it clearer to future devs why these style properties are passed separately. Let me know if you still think it should be removed, though, and I'll be happy to remove it.

Reply:
oh, right - this can be extracted from `imgStyle` - we could/should just reuse that if user decides to overwrite it

Reply:
and use values from there (if set) for `font-family` weird IE hack :)